tests/DCPS/Compiler/keywords/run_test.pl

```diff
@@ -0,0 +1,19 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+    & eval 'exec perl -S $0 $argv:q'
+    if 0;
+
+use strict;
+use warnings;
+
+use lib "$ENV{ACE_ROOT}/bin";
+use lib "$ENV{DDS_ROOT}/bin";
+use PerlDDS::Run_Test;
+
+my $kind = $ARGV[0];
+my $path = "$kind/keywords-$kind";
+die("ERROR: Invalid argument. Pass classic or cpp11.") if (!-f $path);
```

Comment:
See GitHub Actions test failure annotations - I think this is making too many assumptions on what files will exist where.  Maybe just take it out and let the process launching step fail if the argument wasn't correct.